code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -58,24 +58,35 @@
 
 /obj/item/weapon/robot_module/proc/add_languages(var/mob/living/silicon/robot/R)
 	//full set of languages
-	R.add_language("Sol Common", 1)
+	R.add_language("Anglic", 1)
 	R.add_language("Tradeband", 0)
-	R.add_language("Sinta'unathi", 0)
-	R.add_language("Siik'Maas", 0)
-	R.add_language("Skrellian", 0)
 	R.add_language("Gutter", 0)
+	R.add_language("Sinta", 0)
+	R.add_language("Uwe", 0)
+	R.add_language("Weis", 0)
+	R.add_language("Rezar", 0)
+	R.add_language("Zawan", 0)
+	R.add_language("Pekhotha sign", 0)
+	R.add_language("Skrellian", 0)
+	R.add_language("Sini", 0)
+	R.add_language("Kida", 0)
 	R.add_language("Rootspeak", 0)
+	R.add_language("Sign language", 0) //yes, the borg knows sign language. If you need to know why it can use it, a space wizard did it.
+	R.add_language("Zho", 0)
+	R.add_language("Ara", 0)
+	R.add_language("Hindi", 0)
+	R.add_language("", 0)
```

Comment:
I have a fix for robot languages staged. Will try a manual merge on those files.